docs/getting-started/provision-infrastructure.md

```diff
@@ -1,258 +1,333 @@
 ---
 title: Provision Infrastructure
 toc: true
-weight: 3
+weight: 2
 indent: true
 ---
 
-# Provision Infrastructure
-
-Crossplane allows you to provision infrastructure anywhere using the Kubernetes
-API. Once you have [installed a provider] and [configured your credentials], you
-can create any infrastructure currently supported by the provider. Let's start
-by provisioning a database on your provider of choice.
-
-Each provider below offers their own flavor of a managed database. When you
-install a provider it extends Crossplane by adding support for several "managed
-resources". A managed resource is a cluster-scoped Kubernetes custom resource
-that represents an infrastructure object, such as a database instance.
+# Compose Infrastructure
+
+In the [last section] we installed packages that extend Kubernetes with
+infrastructure abstractions and used them to provision resources on a cloud
+provider. These abstractions compose managed resources -- Kubernetes custom
+resources that offer a high fidelity representation of an infrastructure
+primitive, like an SQL instance or a firewall rule. Crossplane goes beyond
+simply modelling infrastructure primitives as custom resources - it enables you
+to define new custom resources with schemas of your choosing. We call these
+"composite resources" (XRs).
+
+XRs are always cluster scoped - they exist outside of any namespace. This allows
+an XR to represent infrastructure that might be consumed from several different
+namespaces. This is often true for VPC networks - an infrastructure operator may
+wish to define a VPC network XR and an SQL instance XR, only the latter of which
+may be managed by application operators. The application operators are
+restricted to their team's namespace, but their SQL instances should all be
+attached to the VPC network that the infrastructure operator manages. Crossplane
+enables scenarios like this  by allowing the infrastructure operator to offer
+their application operators a _composite resource claim_ (XRC). An XRC is a
+namespaced proxy for an XR; the schema of an XRC is identical to that of its
+corresponding XR. When an application operator creates an XRC, a corresponding
+backing XR is created automatically.
+
+We use two special Crossplane resources to define and configure new XRs and
```

Comment:
These resources do not appear in the steps, so I think we can remove this section and rely on composition doc for details.

Reply:
I am kind of torn on this because it feels super critical to explaining the rationale and machinery around cluster-scoping. I agree we are being a little inconsistent in regards to wanting the "magical" experience then doing a fair amount of under the hood explaining here.